requirements/requirements-optionals.txt

```diff
@@ -2,3 +2,4 @@
 markdown==2.5.2
 django-guardian==1.3.2
 django-filter==0.10.0
```

Comment:
Let me phrase that differently - does this affect the test cases if not included?
IE. Is it required for `test_render_tzinfo_object`?